docs/guides/update-cli.md

```diff
@@ -0,0 +1,11 @@
+# Update Redocly CLI
```

Comment:
Add the front matter with `redirectFrom` with the old url `/docs/cli/update-cli/`

Reply:
Got it, thanks!